### DISCUSSION TITLE
Error: Unable to Detect Workspace Platform

### DISCUSSION THREAD

**Discussion by cemalgnlts** [CodinGame/monaco-vscode-api#335] [Q&A]
Hi,

After welcome service override I started getting this error on the console:

```log
Error: Unable to detect workspace platform
gettingStarted.contribution.js:282
```
https://github.com/microsoft/vscode/blob/main/src/vs/workbench/contrib/welcomeGettingStarted/browser/gettingStarted.contribution.ts#L286

This error is not on the demo page. I must have made a mistake somewhere but I am not sure.

**Comment by CGNonofr** [comment]
I think it has to do with the extensionGallery service override being used in the demo

I'm not sure what can be done about it on our side
